### DISCUSSION TITLE
Clearance function

### DISCUSSION THREAD

**Discussion by baaaat** [lairdubois/lairdubois-opencutlist-sketchup-extension#405] [Ideas]
Hi, would it be possible tu give clearance to pieces by using negative value for increase function ?

**Comment by mobilarte** [comment]
What do you mean by clearance function? Please give a screenshot and/or a concrete example showing what you mean.

**Comment by baaaat** [comment]
For example, when I make a mobile shelf, I need a 2mm gap to move the shelf easily. I would like to decrease cutting dimensions!
![Capture d’écran 2022-05-18 à 13 38 45](https://user-images.githubusercontent.com/48209717/169030625-535ad8a2-eaf5-41e2-b5c9-b6577a73cdc2.png)

**Reply by mobilarte** [reply]
Although technically possible, this use case seems to be very dangerous. In your case, simply draw the part to the correct dimension, by removing the clearance you need. OpenCutList should be the least intrusive, in the sense of modifying things in the background as possible, to that what you see is what you get. We consider adding an extra amount around a panel to be safer and it may be useful to woodworker who have their panels pre-cut, but want to do the final squaring.
